webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -164,18 +164,39 @@
                                 <input type="hidden" class='ajax-signature' name="ajax-signature-1" value="{{ crash.signature }}" />
                             </div>
                             <div class="signature-icons">
+                                {% if crash.startup_count > 0 %}
+                                {% if crash.startup_count == crash.count %}
```

Comment:
Why the double if-statement? Why not `{% if crash.startup_count and crash.startup_count == crash.count %}`?